### PR TITLE
Integrating tests

### PR DESCRIPTION
**Pull request by Elexy**
This adds fixtures to the local-transfer tests and runs it for `smallFile` and `largeFile`.
The output parses and gets written to the db.
I also added a more elaborate dashboard.

pending: use schema to align all outputs.

### CODE REVIEW

File: tests/lib/output.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    }
+  }
+}
+
+function validate (data) {
+  const valid = ajv.validate(schema.valueOf(), data)
+  return valid
+}
+
+module.exports = { validate, createFilename, write }
```
Comment (mcollina) [inline tests/lib/output.js]: I think `validate` is better to live together with the schema itself.

File: tests/local-add.js [hunk 2]
```diff
@@ -1,39 +1,46 @@
...
-    return (
-      {
-        name: name,
-        file: file,
-        date: new Date().toISOString(),
+    return build({
+      name: name,
+      subtest: subtest,
+      file: file,
+      date: new Date().toISOString(),
```
Comment (mcollina) [inline tests/local-add.js]: can we automatically pre-fill `date` if it's not passed in?

File: tests/local-add.js [hunk 3]
```diff
@@ -1,39 +1,46 @@
...
-        cpu: os.cpus(),
-        loadAvg: os.loadavg(),
-        memory: os.totalmem() - os.freemem()
-      }
-    )
+        ms: end[1] / 1000000
+      },
+      cpu: os.cpus(),
+      loadAvg: os.loadavg(),
+      memory: os.totalmem() - os.freemem()
```
Comment (mcollina) [inline tests/local-add.js]: I would prefer if we moved this data-grabbing logic within `build` itself.

File: tests/local-add.js (hunk 2 above)
Comment (litzenberger) [reply tests/local-add.js]: this is moved to build - 
https://github.com/ipfs/benchmarks/pull/10

File: tests/local-add.js (hunk 3 above)
Comment (litzenberger) [reply tests/local-add.js]: done! 
https://github.com/ipfs/benchmarks/pull/10

**Pull request merged**